lib/helper/WebDriver.js

```diff
@@ -1604,6 +1604,18 @@ class WebDriver extends Helper {
     }
   }
 
+  /**
+   * {{> type }}
+   * Type out given array of keys or a string of text
+   */
+  async type(keys) {
+    if (Array.isArray(keys)) {
+      await this.browser.keys(keys);
+      return;
+    }
+    await this.browser.keys(keys.split(''));
+  }
```

Comment:
could you add tests? Also do we need to focus on any element to type?

Reply:
Hey @PeterNgTr added the tests, and yes, you will need to 'focus' on the element, but that needs to be done by the tester to imitate the journey "I click on a field and I type.."